client/js/views/Home.vue

```diff
@@ -154,25 +149,66 @@ h1 {
   -webkit-text-fill-color: transparent;
 }
 
-#logo {
-  height: auto;
-  width: 80%;
-  margin: 20px auto;
+h2 {
+	font-size: 1.25rem;
+}
+
+p {
+	margin-top: 1rem;
+}
+
+.container {
+	width: 95%;
+	margin: 0 auto;
+	max-height: 95vh;
 }
 
 .page-title {
   margin: 0 auto;
   text-align: center;
 }
 
-#game-list-card {
-  border-radius: 15px;
-  margin-top: 8px;
+#logo {
+  height: 20vh;
+  margin: 0 auto;
+  .home--large & {
+    height: auto;
+    width: 80%;
+    margin: 20px auto;
```

Comment:
I see we're using `--large` here on all screen sizes, do we just want to make this `.home` instead?